### ISSUE TITLE
DIN EN ISO 10642 added M2 and M2.5

### ISSUE THREAD

**Issue by esopalumpa** [shaise/FreeCAD_FastenersWB#89]
If this site is right, M2 and M2.5 has been added to ISO 10642:
```
Änderungen:
Schrauben aus Edelstahl hinzugefügt
M 2 und M 2,5 hinzugefügt
Bei M14 bis M20 die Gewindelänge bei Schrauben mit Schaft auf 3d erhöht
```
https://shop.dresselhaus.de/de/62-Normen%C3%BCbersicht/inh_id,272,DIN-7991-%3E-DIN-EN-ISO-10642.html

Disclaimer: I have no access to the norm.

**Comment by Gspohu** [comment]
I can confirm there has been added in the ISO 10642:2019.
[https://www.iso.org/standard/72578.html](https://www.iso.org/standard/72578.html)

**Comment by diniciacci** [comment]
I have access to the norm, if someone writes somewhere a small guide on how to add sizes to iso10642def.csv i can do a pull request for it. I had a look but the numbers are unclear to me (also because I am mostly a coder).

**Comment by luzpaz** [comment]
@diniciacci  You can follow the changes that were made in  https://github.com/shaise/FreeCAD_FastenersWB/pull/91/files

**Comment by shaise** [comment]
Added by @alexneufeld.

**Comment by luzpaz** [comment]
Closed by 5b7e17d (thanks @alexneufeld!)
